loculus_values/environment_specific_values/production.yaml

```diff
@@ -18,7 +18,6 @@ secrets:
       preprocessingPipelinePassword: AgBfkwmtZWSX7hcrJYeUwYozsbYycBr0zHWE0RbNPylVugpavVwFdj/K/y3awoGfRIpSIufgO/lTd52Jl9l0TYkd+kyH7LIuBDHD5utV6Aw4tE6CVULx+zESwDxyXhrtQAUBDk2CSpLhzH9fVEJrqAP6YMPG9bF4LtYhHYUUwn62pOMmb2gN7rEitMMxcDD0CKUFczbsVvM84khqgwA0gkehnnld5iLu5M2JLikCXMXTLi9EQ/I9Kkolb5H7hh5Eebuu8rAw+jgyrJhKTbhjz41YS16Qa5+WjJ0bQwU0CYsJB/oNl3xo8L5MdEirlEMXK9gNtp774UNoycpAhU/7a3HrIMcgYVlXtBlXmR5KOYrwz6aaH7Djft1tVKVkYIxnrF4Xs7OQjtH+rbcKDMadBRcYhzhjHM/dlZQ/lJj3MNHo7dCte48CvXuWVnC3E9xo3frl/cFGkBrOp4Ugj4+k4x2aTOkBPV8HgpC75fHGHwucEX+8wrUNfkYJykGlszs6wTgycysKNX0D1xlO+74oNS0QmEtMavDwW80R8lMi038OEL3fnpyKYGg6QHavbk1Hf0SGE7g3HKBuDnJIeIUJiozeaKZzjtzs1dYtMuqw/6XKqjpmJapmYv1om1Bdxq4zqk1ktVUzcIgS6TsgypK2UF+RBLEnWQdSFbVsxSK269XKuH7bEfixSCN5Sn/034aV20aNd0l72ORMzl6Ux1xBVZldXKUJRXkiMw==
       insdcIngestUserPassword: AgBz6FYEAv3dG/zqUdNjq7tvVYNcF5GiNrwRfUm9E7Af4keHzeA4akYJoECaS7L/sjrWw+VwsHAuRJ9aQZldOmX+YSFTd6rwQEYz9ZiGEagxeU6IbIMc2BUmytN/WxhulIcMUhahnWYCPoL9GduJEWdxY7M2P5r3wRSQUi7mBr6acaR1bnBntQRS4GUtoq77jTlBZ7UjzvAH4ZAMC30hRm3Rb9jIHC0Q6Vb7ODKnmTD8AufuQTWPzbAMvsUKzLu3zsYCeVEbdYBAtANNSAP1gk44WZJhTssXDH/stTwEiK01l8tD4xdQRbM+xNHernCwI12ZWgrJcDSCnvRLVUmGF+57NzRqSNMS9t6Q0RFbvYOGBsjhO+qZFTcF43AxoKVpOTqNMe50P93gaYHSpQ905q2JgDNzkWTg3bMbWIDFmZUx6acwOmpTsqochRvAhKU83M1anNKfnw2KGYvsiDNSFZIwhMfeS6gvngavegPT4yrEtPTLJTjjh2tGbYV1cgv0YGvsRdVAwAKejuJ6/OUg7xgmpH/lWpiSggOL7av3wNabQJPsn9JQ0NcaGGyX8ipIpA26SEECyUo+Y5+kGqG8dXdKHGyO/gcCZTxB4VLNnb8xpv3W/i4r1pytiFgh8beZnXZEX+ajWviLff7zTGrkpGUScP8Oa4VwPNPbPhLE9SZaxXq2ulA9OSwMuOjFfqnqT1Nqz6z85ErlbqBp/0UXzk/fW2D6kPw=
-      siloImportJobPassword: AgAYKVrJ4D+zptJjRZ6FcTAtCmPfHd5VdufbeablzGdUymvGmgOjvj05iVXipIdl6Yqkojf/jYAoM2CvK3cNafBSXDKkXSm1lpCvk4Smk5RtVLCCLByBS+O1ch39icXfDyz43VXrOBBCjOfajSC4MYGIOKkzVwr1JbjXJt4V9RZcwQURUqBygZ1hkeTM0bKB3cHN+9jqbpSZPeQ0fAEL5zx0j+gQzpID3qebROy/2WQ/HnNebt8coC4GtOSKpvsgM7JwTzyZLsRhXj5cbVXZNoC4fjYw2rRorZ5FG7h6O5cr9LgST8w8hhd8XoU3oKtRm7b+WwTqeFlXD8o3JwmyCo3HjKYZRY80solA44lGwlA1lJaBrOjbDyG+L4kgC9g2iM4JHAI2w6nEvQEmunjxrIYhQ2PSShMcmbrRqCSzm6KDyC4ctxwbvAQR8MBps77eTypbeJXzp4p9RgrzzhuTrC/6gIjpvjE69295BGLpxKFXfDXJxUNbYtDcAV48ZhLyu3m70ZGvhUPIiPghdxD06L9KcQnlj4TK9AcRzFAvpbT4lzrFrh6pXxIAOKtP5KdvHsSLwJw7eLiGbh4YWmssmZ+onfCBlHAGD9RROkGwoyBPst2ZrcK49hj5Z2FdkvoSoc6TPnVz/N90MigLMYgZ266T1CfN1/J/NwZ9amL+Y5pEdCMqy4jv/wp0YmpafVDK9wsxxBLaw1AJL4l98VjZkSijbek=
```

Comment:
how come?

Reply:
I assume this is because get-released-data is now public

Reply:
Aha - thanks!

Reply:
But in that case let's do it everywhere, e.g. demo